services/web/server/tests/unit/with_dbs/10/test_resource_manager.py

```diff
@@ -414,7 +414,7 @@ async def test_interactive_services_removed_after_logout(
         reraise=True, stop=stop_after_delay(10), wait=wait_fixed(1)
     ):
         with attempt:
-            mocked_director_v2_api["director_v2_core.stop_service"].assert_awaited_with(
+            mocked_director_v2_api["director_v2_core.stop_service"].assert_any_await(
```

Comment:
are you sure about this change?

Reply:
Let me revert it and check what is happening there if it still fails. In theory it should be equivalent, but there might be and edge case here.